<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>842</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Map_button">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>221</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 15px; /* 모서리 둥글기 설정 */
    background-color: rgba(0, 100, 250, 0.5); /* 투명도 50%의 녹색 */
    color: white; /* 글자색 */
    font: bold 14px; /* 폰트 스타일 */
    padding: 6px; /* 패딩 */
}</string>
    </property>
    <property name="text">
     <string>Map</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Road_button">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>260</y>
      <width>221</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 15px; /* 모서리 둥글기 설정 */
    background-color: rgba(0, 100, 250, 0.5); /* 투명도 50%의 녹색 */
    color: white; /* 글자색 */
    font: bold 14px; /* 폰트 스타일 */
    padding: 6px; /* 패딩 */
}</string>
    </property>
    <property name="text">
     <string>Road Guide</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>697</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
